src/platform/backends/qemu/qemu_platform.h

```diff
@@ -42,7 +42,11 @@ class QemuPlatform
     virtual optional<IPAddress> get_ip_for(const std::string& hw_addr) = 0;
     virtual void remove_resources_for(const std::string&) = 0;
     virtual void platform_health_check() = 0;
-    virtual QStringList platform_args(const VirtualMachineDescription& vm_desc)
+    virtual QStringList vmstate_platform_args()
+    {
+        return {};
+    };
+    virtual QStringList vm_platform_args(const VirtualMachineDescription& vm_desc)
     {
         return {};
     };
```

Comment:
I wonder if these should be made pure virtual? To force descendant classes to return empty, but only if that's applicable?

Reply:
Perhaps `vm_platform_args()` should be pure virtual since I would say any platform inheriting this will need to implement this, but I'm not so sure about `vmstate_platform_args()`.  If we really want this to be an interface definition, then *all* methods should be pure virtual.

Reply:
I'm just trying to avoid "forgetting" about those in descendants, is all. Your call :)